### ISSUE TITLE
Time field is Not Empty when using 'Allow Empty Value' and wrapped with [if <time-field>][/if]

### ISSUE THREAD

**Issue by PodsBot** [pods-framework/pods#4171]
Time field is Not Empty when using 'Allow Empty Value' and wrapped with [if <time-field>][/if] submitted via Slack by jimtrue

Test running with Date Field, Date/Time Field and Time Field. This is my template for testing:

```
[if test_date]<div>Date Field: {@test_date}</div>[else]No Date Field[/if]
[if test_date_time]<div>Date Time Field: {@test_date_time}</div>[else]No Date/Time Field[/if]
[if test_time]<div>Time Field: {@test_time}</div>[else]No Time Field[/if]
```

**Comment by JoryHogeveen** [comment]
Same issue as #4145

**Comment by jimtrue** [comment]
Actually that's a date field; my date and date/time fields are fine when they're empty in 2.6.8. Only 'time' has this particular issue.

I'm doing the test first one:

Creating the entry with nothing saved in those fields (empty), time failed that test
Then testing them with data and then deleting the date/time entries

**Comment by jimtrue** [comment]
Actually correction; if FAILS on all three (they should be going into the ELSE condition) because they are all empty, but only TIME displays a value, even though it's empty.

**Comment by jimtrue** [comment]
Time actually saves MIDNIGHT into the field even though 'allow empty' is checked.

**Comment by jimtrue** [comment]
This has been corrected with #4105

**Comment by jimtrue** [comment]
Sorry #4106
